deploy-board/deploy_board/templates/groups/asg_metrics.tmpl

```diff
@@ -36,6 +36,8 @@ var ALARM_WARNING_TEXT = `CPUUtilization > ${ALARM_WARNING_THRESHOLD}% is recomm
                     <div class="col-xs-10">
                         <input class="form-control" name="metricsUrl_{{ alarm.alarmId }}" required="true"
                                 type="text" v-model="metrics"/>
+                        <a href="{{ alarm.cwLink }}" target="_blank">CloudWatch alarm link (fore reference only)</a>
```

Comment:
Did you mean to keep the `"(fore reference only)"` string? Seems it doesn't match the screenshot

Reply:
The screenshot was taken before the change. Fixed the typo though.